Actions/TelemetryHelper.psm1

```diff
@@ -14,7 +14,7 @@ $signals = @{
     "DO0082" = "AL-Go action ran: AnalyzeTests"
```

Comment:
Today all the telemetry functions are used through BcContainerHelper.
We should be able to add to telemetry without containerhelper and then add DO numbers for new actions here,
Will add that as a separate refactoring workitem - we should not hold this PR up for that I think